plugins/twilio/classes/Controller/Sms/Twilio.php

```diff
@@ -52,6 +52,7 @@ public function action_reply()
 		$message_sid  = $this->request->post('MessageSid');
 
 		// Check if a form id is already associated with this data provider
```

Comment:
So this is a setting in Datasources that was added for Comrades you can associate an inbound message for twitter or certain sms providers with a form id, we can extend it to add for others. But it's controlled via a different system which is the dataprovider config which is in a whole other place :/

Seems like it is kind of needed.

Reply:
This is only needed if we enable multiple targeted surveys at the same time (right now we limit it to 1) right? it'd be really good to have